.github/workflows/wiki.yaml

```diff
@@ -1,11 +1,10 @@
 name: Wiki
 on:
   push:
+    branches:
+      - master
```

Comment:
@nerdvegas This will limit us to see if the wiki builds correctly after changes to it. What we could do is allow all branches, but only run the publish step if the current branch is master. What do you think?